nbd-server.c

```diff
@@ -550,7 +558,7 @@ SERVER* cmdline(int argc, char *argv[], struct generic_conf *genconf) {
 	serve=g_new0(SERVER, 1);
 	serve->authname = g_strdup(default_authname);
 	serve->virtstyle=VIRT_IPLIT;
-	while((c=getopt_long(argc, argv, "-C:cdl:mo:rp:M:V", long_options, &i))>=0) {
+	while((c=getopt_long(argc, argv, "-C:cwdl:mo:rp:M:V", long_options, &i))>=0) {
```

Comment:
I have previously NAK'd patches meddling with the command line, because I think starting an export from the command line is bad practice, and that it shouldn't be encouraged. As such, I don't generally accept patches anymore to add more features that can be started from the command line.

I could be convinced otherwise however; can you explain why you want to do this?